agent/whatweb.py

```diff
@@ -0,0 +1,136 @@
+"""WhatWeb Agent : Agent responsible for finger-printing a website."""
+import logging
+import json
+import subprocess
+import tempfile
+import os
+from typing import Union
+from rich import logging as rich_logging
+import io
+
+from ostorlab.agent import agent
+from ostorlab.agent import message as msg
+
+from agent import definitions as whatweb_definitions
+
+logging.basicConfig(
+    format='%(message)s',
+    datefmt='[%X]',
+    handlers=[rich_logging.RichHandler(rich_tracebacks=True)]
+)
```

Comment:
I think you should add : level='INFO', force=True
But please confirm with Alae or Amine.